src/rust/generators/osquery-generator/src/main.rs

```diff
@@ -10,45 +7,19 @@ use rust_proto::graplinc::grapl::{
     },
 };
 use thiserror::Error;
-use tracing_subscriber::{
-    prelude::*,
-    EnvFilter,
-};
 
 mod parsers;
 
 #[non_exhaustive]
 #[derive(Debug, Error)]
-pub enum OsqueryGeneratorError {
-    #[error("error configuring tracing {0}")]
-    TraceError(#[from] opentelemetry::trace::TraceError),
-}
+pub enum OsqueryGeneratorError {}
```

Comment:
this doesn't appear to be used, and it's a little weird to have an export from `main.rs`

Reply:
Sort of. This whole thing is a shambling corpse, but such an enum will be useful once ported to SDK